### ISSUE TITLE
Exposing Matrix Externally (Usage of Alkemio platform identities for Matrix)

### ISSUE THREAD

**Issue by techsmyth** [cherrytwist/Coordination#1039]
## Description
Enabling Alkemio identities in matrix unblocks being able to fully leverage Matrix. 

## Initiative / goal
Innovation Spaces
Communications Flexibility
Matrix Scalability (Matrix Adapter Sync bottleneck)

### Hypothesis
By having other platforms, both external and internal to our cluster, be able to work with Alkemio identities we have a much more flexible identity management infrastructure.

There are numerous benefits / features that will be unblocked by that, like:
- notifications on different channels (e.g. Whatsapp)
- Ory Hydra will enable Alkemio identities to be usable outside of the cluster
- Multi-TLD setup
- Informal messaging on other matrix-based clients, like Element
- Possibility of 3rd-party moderation of alkemio-based data / content

## Acceptance criteria and must have scope
- [ ] Ability to use OIDC identities with Matrix
- [ ] PoC of Ory Hydra working with Ory Kratos
- [ ] Ory Hydra added to k8s infrastructure
- [ ] Evaluate usage of OIDC plugin with Matrix identities

## Stakeholders
DevOps mgr

## Timeline
tbd

## Design
Figma file:
_Comments are highly appreciated. When commenting in Figma, please also leave a note in the comments box below this issue to notify us_

**Comment by techsmyth** [comment]
Closing in favor of #1567
